packages/expo-av/android/src/main/java/expo/modules/av/progress/ProgressLooper.kt

```diff
@@ -0,0 +1,74 @@
+package expo.modules.av.progress
+
+typealias TimeMachineTick = () -> Unit
+
+interface TimeMachine {
+  fun scheduleAt(intervalMillis: Long, callback: TimeMachineTick)
+  val time: Long
+}
+
+/**
+ * @mczernek: NOTE that this class has corresponding test suite. However, since I was unable to set
+ * them up correctly, in order to run them, you should uncomment dependencies in module's build.gradle,
+ * and contents of test class: at packages/expo-av/android/src/test/java/expo/modules/av/progress/ProgressLooperTest.kt
+ * Note, that this breaks client's build, so remember to comment it back.q
+ */
```

Comment:
Is it possible to extract tests to other PR where we can discuss what's wrong with the tests? 🤔

Reply:
I've already solved this issue. It's coming.